release.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  branches: ['master'],
+  plugins: [
+    '@semantic-release/commit-analyzer',
+    '@semantic-release/release-notes-generator',
+    '@semantic-release/changelog',
+    '@semantic-release/npm',
+    '@semantic-release/git',
+  ],
```

Comment:
Appreciate the documentation, editing right now to support releasing shisell from `dist`. 
Why use `['@semantic-release/exec', {prepareCmd: './build.sh'}]` instead of `prepack` command?

Reply:
because the `prepack` command runs after you run `run publish` and before the package is actually packaged. we need to run the `build` script _before_ we run `run publish` because the `run publish` command needs to run inside the `dist` folder that the build script creates